examples/fine-tuning/create_dataset.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+if [ "$#" -ne 2 ] || ! [ -d "$2" ]; then
+  echo "Usage: $0 <NEW_DATASET_DIR> <MNIST_DIR>" >&2
+  exit 1
+fi
+
+NEW_DATASET_DIR=$1
+MNIST_DIR=$2
+
+# create new dataset directories
+mkdir -p $NEW_DATASET_DIR/odd
+mkdir -p $NEW_DATASET_DIR/even
+
+# create symbolic links
+for digit in 0 2 4 6 8
+   do cp $MNIST_DIR/$digit/*.png $NEW_DATASET_DIR/even/
```

Comment:
You could use symlinks here instead of copying the data.

https://github.com/NVIDIA/DIGITS/blob/v3.0.0-rc.3/docs/ImageFolderFormat.md#example-use-case---imagenet-subset

The only concern would be if symlinks aren't supported on an os, but if you're running `/bin/sh` I would expect symlinks to be supported.

May not be worth the effort, just thought I'd mention it in case you didn't know or hadn't thought of it.